src/Foliage.rb

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env ruby
+
+class Foliage
+  @height
+  @offset
+
+  def initialize
```

Comment:
Need documentation for constructor

Reply:
https://github.com/Wall-Dough/pointless/pull/5/commits/8c10c55e48c17c60ff39da532ccd4db52c8fc4b1